numba/np/numpy_support.py

```diff
@@ -481,9 +507,10 @@ def make_specific(outputs, unit):
             try:
                 inputs = choose_types(input_types, ufunc_inputs)
                 outputs = choose_types(output_types, ufunc_outputs)
-                # if the left operand or both are timedeltas, then the output
+                # if the left operand or both are timedeltas, or we have
+                # 1 datetime and 1 timedelta, then the output
```

Comment:
```suggestion
                # 1 datetime and 1 timedelta, then the output
```
the code implies the order is specific but this comment does not?

Reply:
Update the comment. Thank you.